docs/src/main/paradox/docs/delta/api/current/assets/views/deprecate.sh

```diff
@@ -0,0 +1,2 @@
+curl -X DELETE \
+     "https://nexus.example.com/v1/views/myorg/myproj/nxv:myview?rev=5"
```

Comment:
I think it should be `?rev=2` if you follow the examples sequentially.
Same for the elasticsearch deprecate example

Reply:
It is `?rev=3` as there is an update and tagging operations before